websockify.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: websockify
```

Comment:
as this is a python based package can we have the package name as py3-websockify

Reply:
I was thinking the same but noticed that Alpine didn't use py3 prefix. Should we? AFAIK, this is not a library.